pages/api/student_email_join.js

```diff
@@ -2,17 +2,36 @@ import prisma from '../../prisma/prisma';
 
 export default async function handle(req, res) {
   const data = JSON.parse(req.body);
-  //check if the student email exists in the freecodecamp database with their api?
-
-  //add student email to the database
-  const createdStudentEmail = await prisma.classroom.update({
-    where: {
-      classroomId: data['classId'][0]
-    },
-    data: {
-      fccUserIds: { push: data['email'] }
+  const userInfoReq = await fetch(
+    `http://localhost:3002/getstudentprofile?email=${data['email']}`
+  );
+  const userInfo = await userInfoReq.json();
+  if (userInfo.length !== 0) {
+    const existingStudent = await prisma.classroom.findMany({
+      where: {
+        classroomId: data['classId'][0]
+      }
+    });
+    const ids = existingStudent[0]['fccUserIds'];
+    if (ids.includes(userInfo[0]['uuid'])) {
+      return res.status(409).json({
+        error: 1,
+        msg: 'Account already exists'
+      });
```

Comment:
I would consider either using the data (i.e. alerting with `msg`) or simply sending a status code.  Returning the data will make future maintainers either assume that it will be consumed by the caller or have to check the logic and see that it isn't. Neither is great.